### ISSUE TITLE
'sleep_time' is not defined

### ISSUE THREAD

**Issue by emmanuelobo** [Ankit404butfound/PyWhatKit#200]
## Description
When trying to send a WhatsApp group message instantly I am getting the below error. It seems like the variable `sleep_time` should actually be `wait_time`.

```
  File ".../lib/python3.9/site-packages/pywhatkit/whats.py", line 128, in sendwhatmsg_to_group_instantly
    time.sleep(sleep_time)
NameError: name 'sleep_time' is not defined
```
## To Reproduce

Just try sending a group message instantly, the bug should always appear.

## Version Information

- OS: MacOs
- Browser: Safari
- Python: 3.9
- PyWhatKit: 5.3

**Comment by Naman23-coder** [comment]
@emmanuelobo it was fixed in #170, it will be released to pypi shortly

**Comment by emmanuelobo** [comment]
Okay great, didn't notice that. I'll close this out. Thanks.
